src/components/RemoteImage.js

```diff
@@ -40,11 +40,12 @@ class RemoteImage extends React.Component {
       this._img = new Image()
       this._img.addEventListener('load', this.handleLoad)
       this._img.src = src
-      return { exists: false }
+      return { exists: false, loading: true }
```

Comment:
As far as I can tell we don't actually use this state?

Reply:
Yes I was using it at some point, but then I left it because it will be useful in the future (to implement the delay we were talking about yesterday).